src/test/ui/issues/issue-63398.rs

```diff
@@ -0,0 +1,11 @@
+// edition:2018
```

Comment:
Move this to `src/test/ui/async-await/issues/`, include a short description of the problem in the filename, and also please add a comment to the test.